apps/cowswap-frontend/src/legacy/components/Header/AccountElement/index.tsx

```diff
@@ -44,7 +49,15 @@ export function AccountElement({ className, standaloneMode, pendingActivities }:
           </BalanceText>
         )}
         <Web3Status pendingActivities={pendingActivities} onClick={() => account && toggleAccountModal()} />
-        {account && isNotificationsFeedEnabled && <NotificationBell onClick={() => setSidebarOpen(true)} />}
+        {account && isNotificationsFeedEnabled && (
+          <NotificationBell
+            unreadCount={unreadNotificationsCount}
+            onClick={() => {
+              clickNotifications(unreadNotificationsCount ? 'click-bell' : 'click-bell-with-pending-notifications')
```

Comment:
Isn't this inverted?
When the count is truthy there are pending notifications.

Reply:
I forgot to push a commit I have in my local git 🙈